typings/index.d.ts

```diff
@@ -12,17 +12,68 @@ import {
   AndroidNativeProps,
 } from "@react-native-community/datetimepicker";
 
-export type CancelButtonComponent = React.ComponentType<{
-  isDarkModeEnabled: boolean;
-  onPress(): void;
-  label: string;
-}>;
-
-export type ConfirmButtonComponent = React.ComponentType<{
-  isDisabled: boolean;
-  onPress(): void;
-  label: string;
-}>;
+export type CancelButtonStylePropTypes = {
+  button: {
+    borderRadius: number,
+    height: number | string,
+    marginBottom: number | string,
+    justifyContent: string,
+  },
+  buttonLight: {
+    backgroundColor: string,
+  },
+  buttonDark: {
+    backgroundColor: string,
+  },
+  text: {
+    padding: number | string,
+    textAlign: string,
+    color: string,
+    fontSize: number,
+    fontWeight: string,
+    backgroundColor: string,
+  },
+};
+
+export type ConfirmButtonStylePropTypes = {
+  button: {
+    borderTopWidth: number,
+    backgroundColor: string,
+    height: number | string,
+    justifyContent: string,
+  },
+  buttonLight: {
+    borderColor: string,
+  },
+  buttonDark: {
+    borderColor: string,
+  },
+  text: {
+    textAlign: string,
+    color: string,
+    fontSize: number,
+    fontWeight: string,
+    backgroundColor: string,
+  },
+};
+
+export type CancelButtonPropTypes = {
+  isDarkModeEnabled?: boolean,
+  cancelButtonTestID?: string,
+  onPress: () => void,
+  label: string,
+  buttonTextColorIOS?: string,
+  style?: CancelButtonStylePropTypes,
```

Comment:
Same as above, I don't think we're passing this down to the custom cancel/confirm components.

Reply:
Well, I tried changing the `buttonTextColorIOS` and the `style`. Both worked like a charm.

As I said in the previous conversation, all I wanted was to change the button color. But these other properties might be useful for someone in the  future. 

For example: Maybe before **cancel** or **confirm** you want to prompt an Alert.